closed/GensrcJ9JCL.gmk

```diff
@@ -76,7 +80,8 @@ IncludeIfUnsure := -includeIfUnsure -noWarnIncludeIf
 
 $(J9JCL_SOURCES_DONEFILE) : \
 		$(foreach dir, $(JppSourceDirs), $(call RecursiveWildcard,$(dir),*.java)) \
-		$(OPENJDK_STAGED_PROCESS_ENVIRONMENT)
+		$(OPENJDK_STAGED_PROCESS_ENVIRONMENT) \
+		$(OPENJDK_STAGED_SECURITY)
```

Comment:
With the suggested change above, this is simplified to `$(COPY_OVERLAY_FILES)`.

Reply:
done